src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,45 @@
+.nav {
+  &__list {
+    display: flex;
+    flex-direction: row;
+
+    height: 100%;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    box-sizing: border-box;
+    position: relative;
+
+    display: block;
+
+    height: 100%;
+    padding: 24px 0;
+    color: #000;
```

Comment:
Requirements: Nav links color is not black any more.
try to change link color "#000" to "#060b35" to pass a test.
Requirements: Nav links color is not black any more.